packages/editor/src/App.tsx

```diff
@@ -67,7 +62,7 @@ function App(props: AppProps) {
       <div ref={editorRef} className='editor-root' data-theme={theme}>
         <EditorContextInstance.Provider
           value={{
-            pid: props.pid,
+            context: appState.initialData.context,
```

Comment:
Why has this changed from props to appState?

Reply:
because up to now it is still supported that you request a inscription view only with a pid (this is very helpful for testing as I don't want always provide also the app and the pmv).
Then the resolved app and pmv for the given pid is delivered by the server.